wiki/commands-and-permissions/permissions.md

```diff
@@ -4,7 +4,9 @@ description: null
 
 # Permissions
 
-## SuperPerms Nodes
+## SuperPerms Nodes (Auto added)
```

Comment:
Maybe specify that not all super nodes are given.
"(Some are auto added)"

And on the list of super nodes put something like:
simpleclans.anyone.* **(auto added)**

Reply:
Done! And re-wrote the description.